courses/free-courses-zh.md

```diff
@@ -15,6 +15,7 @@
 
 ### AI
 
+* [动手实战人工智能](https://aibydoing.com) - huhuhang
```

Comment:
This looks like a textbook for a course rather than a course. Is it? If so it should be on the books list